docs/pages/project/contributing/contributing-model-relationships.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: page
```

Comment:
@vishalvivekm Should this be `enhanced`?

Reply:
we are actually using page layout for all contribution pages

Reply:
Thank you